reggie/configs/data/arizona.yaml

```diff
@@ -35,6 +35,10 @@ voter_status_suspense: 'S   '
 voter_status_cancelled: 'R   '
 voter_status_not_eligible: 'N   '
 voter_status_not_registered: 'NR  '
+not_registered_voter_status_set:
+  - 'R   '
```

Comment:
It seems inline with the values above, but this white space seems problematic.  Can we not just do a "R" and then the actual logic strips white space before comparison?

Reply:
For most string values, we do the whitespace stripping and a bunch of other normalization (lowercasing, etc). However, the voter status and party values were always designated literally in the yamls, so we can't do that normalization on those strings without rewriting all the yamls and a bunch of code. I think this is one of those things we just fix in the rewrite :) Also, this is for Arizona not Arizona2, so I really only even changed it here as breadcrumbs for us in the future.